cloud/jenkins/psmdb_docker_build.groovy

```diff
@@ -31,15 +31,15 @@ void checkImageForDocker(String IMAGE_PREFIX){
     }
 }
 
-void pushImageToDocker(String IMAGE_PREFIX){
+void pushImageToDocker(String IMAGE_SUFFIX){
     withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
         sh """
-            IMAGE_PREFIX=${IMAGE_PREFIX}
+            IMAGE_SUFFIX=${IMAGE_SUFFIX}
             sg docker -c "
                 docker login -u '${USER}' -p '${PASS}'
                 export DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE="${DOCKER_REPOSITORY_PASSPHRASE}"
-                docker trust sign perconalab/percona-server-mongodb-operator-${IMAGE_PREFIX}
-                docker push perconalab/percona-server-mongodb-operator-${IMAGE_PREFIX}
+                docker trust sign perconalab/percona-server-mongodb-operator:master-${IMAGE_SUFFIX}
+                docker push perconalab/percona-server-mongodb-operator:master-${IMAGE_SUFFIX}
```

Comment:
@delgod if we are not going to change it right now in `percona-server-mongodb-operator` repository let's change that later when everything changes

Reply:
@delgod ah, but this is image name, did you mean to rename just the images on the dockerhub? in that case I misunderstood?

Reply:
@hors ok, but I propose we leave it as `master` here for now until we go with changing everything, since we might end up in a strange situation and confusion until git branch is renamed.